pkg/trimaran/handler_test.go

```diff
@@ -75,3 +76,71 @@ func TestHandlerCacheCleanup(t *testing.T) {
 		})
 	}
 }
+
+func TestHandlerCacheCleanup(t *testing.T) {
+	now := time.Now()
+	tests := []struct {
+		name               string
+		scheduledPodsCache map[string][]podInfo
+		expectedCache      map[string][]podInfo
+	}{
+		{
+			// 没有过期pod
```

Comment:
could you update the comments in English?

Reply:
@Huang-Wei  Already deleted